pdfminer/converter.py

```diff
@@ -563,8 +565,52 @@ def render(item):
                     '<textline bbox="%s">\n' %
                     bbox2str(
                         item.bbox))
-                for child in item:
-                    render(child)
+                if self.coordinates_type == 'c':  # Output for every character
+                    for child in item:
+                        render(child)
+                elif self.coordinates_type == 'w':  # Output for every word
+                    coord = [0, 0, 0, 0]
```

Comment:
It would be nice with some more comments or documentation on what is happening here. It could be helpful by putting the code after each elif into a seperate method so that it is easier to test and understand the code.